tests/dragonfly/replication_test.py

```diff
@@ -3171,3 +3170,76 @@ async def test_replicate_hset_with_expiry(df_factory: DflyInstanceFactory):
 
     assert "name" in result
     assert result["name"] == "1234"
+
+
+@pytest.mark.parametrize("proactors", [1, 4, 6])
+@pytest.mark.parametrize("backlog_len", [1, 256, 1024, 1300])
+async def test_partial_sync(df_factory, df_seeder_factory, proactors, backlog_len):
+    keys = 5_000
+    if proactors > 1:
+        keys = 10_000
+
+    master = df_factory.create(
+        proactor_threads=proactors, shard_repl_backlog_len=backlog_len, lock_on_hashtags=True
```

Comment:
cool, please add a comment why we use the lock_on_hashtags and the prefix in the set command so it will be clear to everyone

Reply:
it is clear to me what the flag is doing but without the comment it would not be clear what is the intent.